torch/csrc/jit/codegen/cuda/transform_iter.cpp

```diff
@@ -107,15 +131,15 @@ void ReplayTransformations::handle(Merge* m) {
 ReplayTransformations::ReplayTransformations(
     const std::vector<IterDomain*>& _target_domain,
     std::unordered_map<IterDomain*, IterDomain*> _id_map,
-    bool _check_all_ops_run)
+    bool _error_on_failure)
```

Comment:
why the leading underscore?

Reply:
just a habit, I've been doing this on constructors when I copy it directly to a member named with a trailing underscore, is it bothersome?

Reply:
naming patterns are a powerful tool/hint, best when used sparingly (otherwise you end up with https://en.wikipedia.org/wiki/Hungarian_notation)

If we have a good reason to adopt a _ prefix we could, but then we'd need to use it consistently.

Reply:
I personally liked this for Constructor parameters.   I thought this was being used consistently.

Reply:
I have been trying to be consistent about this and really like it as clang tidy tends to force us to use std::move which helps me remember not to use things with a prefix except in initialization. We can discuss further but thinking to leave it for this PR.